clients/rospy/src/rospy/client.py

```diff
@@ -289,8 +286,7 @@ def init_node(name, argv=None, anonymous=False, log_level=None, disable_rostime=
     # check for name override
     mappings = rospy.names.get_mappings()
     if '__name' in mappings:
-        # use rosgraph version of resolve_name to avoid remapping
-        name = rosgraph.names.resolve_name(mappings['__name'], rospy.core.get_caller_id())
```

Comment:
So that is true for `rospy.names.resolve_name` https://github.com/ros/ros_comm/blob/indigo-devel/clients/rospy/src/rospy/names.py#L163-L178

But not for `rosgraph.names.resolve_name` https://github.com/ros/ros_comm/blob/indigo-devel/tools/rosgraph/src/rosgraph/names.py#L260-L275